lib/console_display.rb

```diff
@@ -25,41 +29,55 @@ def show_board
   end
 
   def play_round
-    current_player = @game.calculate_current_player
-    current_move = current_player.set_current_move
-    play_empty_position(@game, current_move, current_player)
-    show_board
+    play_empty_position
   end
 
   def format_board
-    rows = create_rows
     display = ROW_SEPARATOR
-    display += " | " + rows.flat_map { |row|
-      [row, ROW_SEPARATOR]}.join(" | ")
+    display += COLUMN_SEPARATOR + rows.flat_map { |row|
+      [row, ROW_SEPARATOR]}.join(COLUMN_SEPARATOR)
   end
 
   private
 
-  def create_rows
+  def rows
     @game.board.cells.each_slice(3)
```

Comment:
this shouldn't know the structure of the board internal data. You are exposing the fact that a) it's an array, and b) it has 3 rows. I would delegate that to the board, so this comes `@game.board.rows`.